[2024-08-16T14:35:05.602+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-16T14:35:05.621+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: third_dag.third_task manual__2024-08-16T14:35:02.351041+00:00 [queued]>
[2024-08-16T14:35:05.630+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: third_dag.third_task manual__2024-08-16T14:35:02.351041+00:00 [queued]>
[2024-08-16T14:35:05.631+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 6
[2024-08-16T14:35:05.650+0000] {taskinstance.py:2879} INFO - Executing <Task(BashOperator): third_task> on 2024-08-16 14:35:02.351041+00:00
[2024-08-16T14:35:05.658+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1076) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-16T14:35:05.659+0000] {standard_task_runner.py:72} INFO - Started process 1077 to run task
[2024-08-16T14:35:05.658+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'third_dag', 'third_task', 'manual__2024-08-16T14:35:02.351041+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpzxa_u20k']
[2024-08-16T14:35:05.660+0000] {standard_task_runner.py:105} INFO - Job 14: Subtask third_task
[2024-08-16T14:35:05.710+0000] {task_command.py:467} INFO - Running <TaskInstance: third_dag.third_task manual__2024-08-16T14:35:02.351041+00:00 [running]> on host 93607a8c7584
[2024-08-16T14:35:05.792+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Juancito2085' AIRFLOW_CTX_DAG_ID='third_dag' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2024-08-16T14:35:02.351041+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-16T14:35:02.351041+00:00'
[2024-08-16T14:35:05.793+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-16T14:35:05.809+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-08-16T14:35:05.810+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo task 3 completed']
[2024-08-16T14:35:05.818+0000] {subprocess.py:86} INFO - Output:
[2024-08-16T14:35:05.820+0000] {subprocess.py:93} INFO - task 3 completed
[2024-08-16T14:35:05.820+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-08-16T14:35:05.845+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-16T14:35:05.846+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=third_dag, task_id=third_task, run_id=manual__2024-08-16T14:35:02.351041+00:00, execution_date=20240816T143502, start_date=20240816T143505, end_date=20240816T143505
[2024-08-16T14:35:05.875+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-16T14:35:05.893+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-16T14:35:05.896+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
